ledfx/utils.py

```diff
@@ -1236,3 +1237,30 @@ def open_gif(gif_path):
     except Exception as e:
         _LOGGER.error(f"Failed to open gif : {gif_path} : {e}")
         return None
+
+
+def get_mono_font(size):
```

Comment:
Can you add some simple docsstrings to these new functions?

Reply:
Yep